code/game/objects/structures/icemoon/cave_entrance.dm

```diff
@@ -965,49 +965,6 @@ GLOBAL_LIST_INIT(ore_probability, list(
 			new /obj/effect/mob_spawn/human/scientist(loc)
 			new /turf/open/floor/mineral/titanium/purple(loc)
 			new /mob/living/simple_animal/slime/random(loc)
-		if(19)//lost abductor
```

Comment:
Need to adjust the numbers if you remove one

Reply:
Should be fixed now. God these other entries suck though.